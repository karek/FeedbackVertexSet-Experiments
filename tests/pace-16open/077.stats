113,161,9
