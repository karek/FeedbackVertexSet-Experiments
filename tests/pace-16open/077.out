16
1 94 4 77 62 69 43 101 73 30 76 109 79 78 81 100 
